docs/configuration/services-configuration.rst

```diff
@@ -147,6 +154,8 @@ DataPublishers
      - The name of the data publisher.
    * - Topic
      - The topic on which the data publisher publishes its information. (optional)
+   * - Labels
+     - The labels determining matching subscribers with the same topic and media type. (optional)
```

Comment:
Although the `Labels` list follows the usual way that *configured* values take precedence over *programmatically* set values, I think it helps to stress this again. (same for Subscriber/RPC docs below)

```suggestion
     - The labels determining matching subscribers with the same topic and media type. (optional)
        Note that these labels will replace all programmatically provided labels.
```